lxl-web/src/lib/components/DecoratedData.svelte

```diff
@@ -184,7 +186,9 @@
 					/>
 					{#if remainder && Array.isArray(remainder)}
 						<span
-							use:resourcePopover={{ data: remainder, lang: getSupportedLocale($page.params.lang) }}
+							use:popover={{
```

Comment:
...Similarly, I'd say the `remainder` stuff in DecoratedData is purely a leftover from the previous card. It should probably just be removed. Maybe not in scope of this PR...

Reply:
...if we don't decide to use it again to prevent [this](https://beta.libris-qa.kb.se/find?_q=musikerminnen+svenska+musikers&_limit=20&_offset=0&_sort=&_i=musikerminnen+svenska+musikers&_x=advanced)

Reply:
Yeah, I was split between keeping or removing `remainder` (as we will need some support for truncating the decorated data) but if the list of items is too long it still becomes an issue even with the current remainder implementation tooltip (see attached screenshot).

<img width="1168" alt="Skärmavbild 2024-07-08 kl  11 39 37" src="https://github.com/libris/lxlviewer/assets/10220360/3a0dcbc0-3912-47e2-82a6-d195123c3a5f">

So, I think I'm currently in favour of keeping it as is for the sake of the scope of this PR. Or should we remove it altogether to later reimplement it? 🤷